components/Map/Map.tsx

```diff
@@ -41,7 +41,7 @@ export const Map = ({
   const center = getCenter(coords)
 
   return loadError ? (
-    <Text>{loadError}</Text>
+    <Text>{loadError.message}</Text>
```

Comment:
Upgraded React types didn't allow an `Error` object as a child here.

Reply:
I'm surprised this worked at any point, I thought you couldn't pass objects as text like that. Good catch though